src/main/jbake/content/documentation/bundles/models.md

```diff
@@ -55,6 +57,35 @@ Constructor injection is also supported (as of Sling Models 1.1.0):
 
 Because the name of a constructor argument parameter cannot be detected via the Java Reflection API a `@Named` annotation is mandatory for injectors that require a name for resolving the injection.
 
+Constructors may use any visibility modifier (as of Sling Models 1.5.0):
```

Comment:
Shouldn't we mention fields and methods here as well for completion? IMHO those never had any restrictions in terms of visibility....

Reply:
For fields, yes. Call me crazy, but while I was digging in, to try to replace reflection with method handles, I did not see method _injection_ anywhere. Everywhere I see method-injection-like code, it only worked with an interface, using the proxy. I am not sure if private methods in interfaces are proxyable, thus injectable.

I would be happy to be proven wrong, though.

Reply:
You are right with regards to methods. Let's just mention fields for now as well.

Reply:
Will make the change and push a bit later today. Regarding method injection for concrete models - I could see that as an enhancement. I'll add a new ticket for that feature

Also, I think the code could use a little attention in `ModeAdapterlFactory`(I REALLY want to refactor it, but just need the time to sit down and understand everything).

Reply:
Updated with a lot more detail re: field and method injection